doc/examples/rendering/traceSetsArnold.gfr

```diff
@@ -0,0 +1,348 @@
+__children["Backdrop3"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -0.707069397, -25.2443771 ), imath.V2f( 39.2929306, 12.5390015 ) ) )
+__children["Backdrop3"]["__uiPosition"].setValue( imath.V2f( -93.0875244, -102.858772 ) )
+__children["RaySwitchShader"]["parameters"]["camera"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["parameters"]["shadow"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["parameters"]["diffuse_reflection"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["parameters"]["diffuse_transmission"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["parameters"]["specular_reflection"].setInput( __children["TraceSet1"]["out"] )
+__children["RaySwitchShader"]["parameters"]["specular_transmission"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["parameters"]["volume"].setInput( __children["StandardSurface"]["out"] )
+__children["RaySwitchShader"]["__uiPosition"].setValue( imath.V2f( -9.90797043, -103.52475 ) )
+__children["ShaderAssignment"]["enabled"].setValue( False )
+__children["ShaderAssignment"]["in"].setInput( __children["ShaderAssignment1"]["out"] )
+__children["ShaderAssignment"]["filter"].setInput( __children["PathFilter"]["out"] )
+__children["ShaderAssignment"]["shader"].setInput( __children["RaySwitchShader"]["out"] )
+__children["ShaderAssignment"]["__contextCompatibility"].setValue( False )
+__children["ShaderAssignment"]["__uiPosition"].setValue( imath.V2f( 7.23167801, -103.52475 ) )
+__children["PathFilter"]["paths"].setValue( IECore.StringVectorData( [ '/world/geo/plane' ] ) )
+__children["PathFilter"]["__uiPosition"].setValue( imath.V2f( 18.0980415, -97.9078064 ) )
+__children["StandardSurface"]["parameters"]["base"].setValue( 1.0 )
+__children["StandardSurface"]["parameters"]["diffuse_roughness"].setValue( 1.0 )
+__children["StandardSurface"]["parameters"]["specular"].setValue( 0.0 )
+__children["StandardSurface"]["__uiPosition"].setValue( imath.V2f( -39.750576, -99.9247513 ) )
+__children["StandardSurface2"]["parameters"]["base"].setValue( 1.0 )
+__children["StandardSurface2"]["parameters"]["diffuse_roughness"].setValue( 1.0 )
+__children["StandardSurface2"]["parameters"]["specular"].setValue( 0.0 )
+__children["StandardSurface2"]["__uiPosition"].setValue( imath.V2f( -39.509861, -115.691742 ) )
+__children["TraceSet1"]["parameters"]["passthrough"].setInput( __children["StandardSurface2"]["out"] )
+__children["TraceSet1"]["parameters"]["trace_set"].setValue( 'spheres' )
+__children["TraceSet1"]["parameters"]["inclusive"].setValue( False )
+__children["TraceSet1"]["__uiPosition"].setValue( imath.V2f( -24.7410622, -115.691742 ) )
+Gaffer.Metadata.registerValue( __children["Backdrop4"], 'nodeGadget:color', imath.Color3f( 0.347900003, 0.438600004, 0.5 ) )
+__children["Backdrop4"]["title"].setValue( 'Note: New objects' )
```

Comment:
### Backdrop title
Note: Assigning sets to new scene locations

### Comment
I've skirted a fine line before by omitting the fact that lights aren't objects, but here we should be careful to remind users that it's the location that matters.

Reply:
I've gone for a shorter title as otherwise its crazy wide.